.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
```

Comment:
For go files indent is always tab.

Reply:
I use 4 spaces in this file. Do I need to change on tab?

Reply:
If my understanding is right - yes and no.
Why yes: `[*]` applies rules to all files in the repository, it means that all files will be formatted using 4 spaces indentation, correct me if I'm wrong. For go files spaces are not flavored indentation. 
Why no: Some files like python scripts may require spaces as indentation, so tabs in this rule will not fit them.
Possible solution: define the rule for any type of file explicitly.

Reply:
I did not understand which values are correct. That is why I deleted .editorconfig file